components/expression/expression-renderer.js

```diff
@@ -265,161 +318,161 @@ ExpressionRenderer.prototype.getHeatmapLabelSizes_ = function() {
     this.geneLabelWidth_ = 30;
   }
   if (!this.data.options.showConditionLabels) {
-    this.conditionLabelHeight_ = 0;
+    this.conditionLabelHeight_ = 100;
```

Comment:
This should be zero. If showConditionLabels option is set to false, then conditionLabelHeight_ must be ZERO.